vulcan/_message.py

```diff
@@ -82,3 +82,9 @@ def get(cls, api, date_from=None, date_to=None):
                 message["NadawcaId"]
             )
             yield to_model(cls, message)
+    @classmethod
+    def send(api, title, content, recipient_id, recipients):
+        for students in api.get_students():
```

Comment:
Don't do that, please add to the `Student` class `account_name` attribute with key `"UzytkownikNazwa"` and use here `api.student.account_name` instead of getting the last student name.